src/services/api/utils.ts

```diff
@@ -2,13 +2,13 @@ import { enableWalletOnce, useMyAccount } from '@/stores/my-account'
 import { Signer, signMessage } from '@/utils/account'
 import type { Signer as InjectedSigner } from '@polkadot/api/types'
 import { toSubsocialAddress } from '@subsocial/utils'
-import { sortObj } from 'jsonabc'
+import sortKeys from 'sort-keys-recursive'
```

Comment:
Can you explain why this one is better?

Reply:
I used the same package used in datahub, and from max's reason he said

because jsonabc has problems with typescript
sort-keys-recursive is just a wrapper of jsonabc but with some typescript fixes
it's error abour cannot use requier()

Reply:
anyway its also only 1kb so no bundle increase